.github/workflows/build.yaml

```diff
@@ -20,16 +20,18 @@ jobs:
             -   name: "Checkout"
                 uses: actions/checkout@v2
 
-            -   name: "Cache dependencies installed with composer"
-                uses: actions/cache@v1
+            -   name: "Install PHP with extensions"
+                uses: "shivammathur/setup-php@v1"
```

Comment:
What do you think about migrating setup-php action to v2 https://github.com/shivammathur/setup-php/wiki/Switch-to-v2?

Reply:
Yes we just need to change the version to v2 IMO